doc/releases/migration-guide-4.0.rst

```diff
@@ -404,6 +404,10 @@ Bluetooth Audio
 Bluetooth Host
 ==============
 
+* :c:func:`bt_gatt_subscribe` behavior has changed to if a different user/application has already
+  subscribed to the same handle, then the callback will contain the values aggregated of all the
+  users/applications.
```

Comment:
This change also needs to added to the documentation of `bt_gatt_subscribe` if not already

Reply:
@Yagoor please be careful about resolving review comments yourself: https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#workflow-suggestions-that-help-reviewers

> Unless they applied the reviewer’s recommendation exactly, authors must not resolve and hide comments, they must let the initial reviewer do it.